pmd-modelica/src/main/resources/category/modelica/misc.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+
+<ruleset name="Misc Modelica rules"
```

Comment:
Actually, there is no "misc" category. This ruleset should probably live under tests (somewhere under src/test/resources).